sherpa-onnx/python/csrc/online-stream.cc

```diff
@@ -34,6 +51,9 @@ void PybindOnlineStream(py::module *m) {
           py::arg("sample_rate"), py::arg("waveform"), kAcceptWaveformUsage,
           py::call_guard<py::gil_scoped_release>())
       .def("input_finished", &PyClass::InputFinished,
+           py::call_guard<py::gil_scoped_release>())
+      .def("get_frames", &PyClass::GetFrames,
+           py::arg("frame_index"), py::arg("n"),
```

Comment:
```suggestion
           py::arg("frame_index"), py::arg("n"), kGetFramesUsage
```

so that if you use `help(OnlineStream.get_frames)`, you can view the help info in python.